src/main/java/org/toradocu/extractor/JavadocExtractor.java

```diff
@@ -428,6 +428,17 @@ private Boolean isNullable(com.github.javaparser.ast.body.Parameter parameter) {
       definitionOpt = cu.getEnumByName(typeName);
     }
 
+    if (!definitionOpt.isPresent()) {
```

Comment:
Conceptually this code belongs to the else branch of the if condition at line 431/442. 
I think is better to move the code there:
```java
if (definitionOpt.isPresent()) {
  ... // code already present
} else {
  ... // your new code
}
```

Reply:
OK: moved